fortios/Makefile

```diff
@@ -0,0 +1,20 @@
+VENDOR=Fortigate
+IMAGE_FORMAT=qcow2
+IMAGE_GLOB=*.qcow2
+-include ../makefile-sanity.include
+-include ../makefile.include
+IMAGE=$(shell find . -type f -iname *.qcow2 | head -n1 | cut -c3- )
+VERSION=$(shell echo $(IMAGE) | sed 's|^fortios-||' | sed -e 's/\.qcow2//')
+print-all:
+	@echo "$(VERSION)"
+	@echo "$(IMAGE)"
+docker-build-fortigate: 
+	-rm -f docker/$(IMAGE) docker/healthcheck.py docker/vrnetlab.py
+	cp $(IMAGE) docker
+	cp ../common/* docker/
+	@echo "Building docker image using $(IMAGE) as $(REGISTRY)vr-$(VR_NAME):$(VERSION)"
+	(cd docker; docker build --build-arg IMAGE=$(IMAGE) --build-arg http_proxy=$(http_proxy) --build-arg https_proxy=$(https_proxy) -t $(REGISTRY)vr-$(VR_NAME):$(VERSION) .)
+	-rm -f docker/$(IMAGE) docker/healthcheck.py docker/vrnetlab.py
```

Comment:
I am not sure this target is actually needed.
if I run just `make` it builds the image just fine

```
❯ make
for IMAGE in fortios-v0.7.0.qcow2; do \
        echo "Making $IMAGE"; \
        make IMAGE=$IMAGE docker-build; \
done
Making fortios-v0.7.0.qcow2
make[1]: Entering directory '/root/hellt/vrnetlab/fortios'
rm -f docker/*.qcow2* docker/*.tgz* docker/*.vmdk* docker/*.iso
Building docker image using fortios-v0.7.0.qcow2 as vrnetlab/vr-fortios:v0.7.0
cp ../common/* docker/
make IMAGE=$IMAGE docker-build-image-copy
make[2]: Entering directory '/root/hellt/vrnetlab/fortios'
cp fortios-v0.7.0.qcow2* docker/
make[2]: Leaving directory '/root/hellt/vrnetlab/fortios'
(cd docker; docker build --build-arg http_proxy= --build-arg https_proxy= --build-arg IMAGE=fortios-v0.7.0.qcow2 -t vrnetlab/vr-fortios:v0.7.0 .)
[+] Building 78.6s (9/9) FINISHED    
```

Reply:
You are right. 
I thougth the first goal would be the default target.
But I would still add
`
.DEFAULT_GOAL := docker-build-fortigate
`
just to be sure this is always the case.
I will also remove `print-all` in the next commit. Was just for debugging purpose

Reply:
You are right. 
I thougth the first target would be default.

Would still add 
`
.DEFAULT_GOAL := docker-build-fortigate
`

the be absolutely sure about it.

Also will remove the `print-all` target since it was just for debugging purposes